pkg/components/bindings/registry.go

```diff
@@ -31,11 +31,7 @@ type Registry struct {
 }
 
 // DefaultRegistry is the singleton with the registry.
-var DefaultRegistry *Registry
-
-func init() {
-	DefaultRegistry = NewRegistry()
```

Comment:
@ItalyPaleAle  Is this OK to move out of init()

Reply:
It was previously aligned offline with @ItalyPaleAle